src/test/java/com/apextrio/peak/PeakApplicationTests.java

```diff
@@ -1,17 +1,319 @@
+	public void testTeamsId() {
+		try {
+			MvcResult response = this.mvc.perform(get("/teams/6")).andExpect(status().isOk()).andExpect(view().name("team")).andExpect(model().attribute("inGroup", false)).andReturn();
+			assertTrue(response.getResponse().getContentAsString().contains("<div id=\"users\">"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testTeamsIdInGroup() {
+		try {
+			this.mvc.perform(get("/teams/1")).andExpect(status().isOk()).andExpect(view().name("team")).andExpect(model().attribute("inGroup", true));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testTeamsIdNoAuth() {
+		try {
+			MvcResult response = this.mvc.perform(get("/teams/3")).andExpect(status().isOk()).andExpect(view().name("team")).andExpect(model().attribute("inGroup", null)).andReturn();
+			assertFalse(response.getResponse().getContentAsString().contains("<div id=\"users\">"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testTeamsIdInvalidId() {
+		try {
+			this.mvc.perform(get("/teams/10000")).andExpect(status().isNotFound()).andExpect(view().name("error"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testLogin() {
+		try {
+			this.mvc.perform(get("/login")).andExpect(status().isOk()).andExpect(view().name("login"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testSignup() {
+		try {
+			this.mvc.perform(get("/signup")).andExpect(status().isOk()).andExpect(view().name("sign_up"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testNewTeam() {
+		try {
+			this.mvc.perform(get("/newteam?resortId=2")).andExpect(status().isOk()).andExpect(view().name("teamForm"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testNewTeamNoAuth() {
+		try {
+			this.mvc.perform(get("/newteam?resortId=2")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("http://localhost/login"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testMyProfile() {
+		try {
+			this.mvc.perform(get("/myProfile")).andExpect(status().isOk()).andExpect(view().name("my_profile"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testMyProfileNoAuth() {
+		try {
+			this.mvc.perform(get("/myProfile")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("http://localhost/login"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	//---------------------------------
+	//TESTING FOR APPLICATION ENDPOINTS
+	//---------------------------------
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testPostResortsId() {
```

Comment:
I might have a few more POST tests where I leave out some params, or add in extra params, or try to do other things that would conflict with how your database is set up.